src/test/java/uk/gov/companieshouse/psc/delta/mapper/PscMapperTest.java

```diff
@@ -0,0 +1,255 @@
+package uk.gov.companieshouse.psc.delta.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.io.InputStreamReader;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.util.FileCopyUtils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.gov.companieshouse.api.delta.NameElements;
+import uk.gov.companieshouse.api.delta.Psc;
+import uk.gov.companieshouse.api.delta.PscDelta;
+import uk.gov.companieshouse.api.psc.Address;
+import uk.gov.companieshouse.api.psc.Data;
+import uk.gov.companieshouse.api.psc.DateOfBirth;
+import uk.gov.companieshouse.api.psc.ExternalData;
+import uk.gov.companieshouse.api.psc.FullRecordCompanyPSCApi;
+import uk.gov.companieshouse.api.psc.ItemLinkTypes;
+import uk.gov.companieshouse.api.psc.SensitiveData;
+import uk.gov.companieshouse.api.psc.UsualResidentialAddress;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {
+        PscMapperImpl.class})
+public class PscMapperTest {
```

Comment:
You've done individual, corp and super secure. I think you should have tests for legal-entity, and one beneficial owner type as well, maybe individual BO.